README.md

```diff
@@ -46,13 +46,14 @@ const fourthWord = "cheese";
 ```
 Result should be:
 ```js
+console.log("Please"+" "+"squeeze"+" "+"the"+" "+"cheese")
```

Comment:
This is not the place to write the answer, please use variables instead of strings for the answer.